tests/Tests/Ingest/ProcessorAssertions.cs

```diff
@@ -59,19 +60,41 @@ public static IPromise<IList<IProcessor>> Fluent(ProcessorsDescriptor d)
 
 		public class Append : ProcessorAssertion
 		{
-			public override Func<ProcessorsDescriptor, IPromise<IList<IProcessor>>> Fluent => d => d
-				.Append<Project>(a => a
-					.Field(p => p.State)
-					.Value(StateOfBeing.Stable, StateOfBeing.VeryActive)
-				);
+			public override Func<ProcessorsDescriptor, IPromise<IList<IProcessor>>> Fluent =>
+				d => d.Append<Project>(a =>
+				{
+					var apd = a.Field(p => p.State).Value(StateOfBeing.Stable, StateOfBeing.VeryActive);
 
-			public override IProcessor Initializer => new AppendProcessor
+					if (TestClient.Configuration.InRange(">=7.11.0"))
+						apd.AllowDuplicates(false);
+
+					return apd;
+				});
+
+			public override IProcessor Initializer
 			{
-				Field = "state",
-				Value = new object[] { StateOfBeing.Stable, StateOfBeing.VeryActive }
-			};
+				get
+				{
+					var ap = new AppendProcessor { Field = "state", Value = new object[] { StateOfBeing.Stable, StateOfBeing.VeryActive } };
+
+					if (TestClient.Configuration.InRange(">=7.11.0"))
```

Comment:
Better to add a `SkipVersion` on the `Append` class and include the additional property. 

This way on master we can clean up the `SkipVersion` easier when we go over all the unnecessary skips.

Reply:
I've done as suggested in the comment and added a separate test which covers allow duplicates on 7.11+